components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/User.java

```diff
@@ -206,12 +209,24 @@ public String toFullQualifiedUsername() {
         return username;
     }
 
+    /**
+     * This method will retrieve the 'CaseInsensitiveUsername' property from the respective userstore and set that value.
+     */
+    protected void setUserStoreCaseSensitive() {
```

Comment:
this method name is bit confusing. It gives the meaning that we are setting userstore case sensitive. Instead, how about something like 'updateCaseSensitivity()' as the username is inferred by the class itself?

Reply:
Yes +1, updateCaseSensitivity() sounds more appropriate.
I think setCaseSensitivity() also sounds good.

Reply:
setX() usually sounds like a setter where it expects a value as a param. So I'd prefer to have updateCaseSensitivity() or something similar.